modules/claims_api/lib/claims_api/bgs_claim_status_mapper.rb

```diff
@@ -46,7 +47,7 @@ def name
     end
 
     def name_from_phase
-      return 'no status received' if @phase_type.nil? || @phase_type.strip.empty?
+      return 'no status received' if @phase_type.nil?
```

Comment:
Note I removed the latter portion of the check here, as the @phase_type variable is an integer not a string, and rspec tests I created were failing with undefined method errors.

Reply:
I think that's fine, as it appears to only be called form [this line](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb#L202), and that's got a `to_i`.